artemis-core-client/src/test/java/org/apache/activemq/artemis/util/CompressionUtilTest.java

```diff
@@ -104,42 +103,6 @@ public void testDeflaterReader2() throws Exception {
       compareByteArray(allCompressed, output, compressedDataLength);
    }
 
-   @Test
```

Comment:
what happened to this test?

Reply:
`InflaterReader` was no longer needed, so could be removed (need to be sure of this one yet)!